views/templates/hook/blocknewsletter.tpl

```diff
@@ -35,6 +35,7 @@
 			<p>
 				<input class="inputNew" id="newsletter-input" type="text" name="email" size="18" value="{if isset($value) && $value}{$value}{else}{l s='your e-mail' mod='blocknewsletter'}{/if}" />
 				<input type="submit" value="ok" class="button_mini" name="submitNewsletter" />
+				{hook h='displayGDPRConsent' mod='psgdpr' id_module=$id_module}
```

Comment:
What if another (native or community) module wants to use this hook? Do you really think that's a good idea to limit that call to a specific module?